package/kernel/linux/modules/other.mk

```diff
@@ -64,6 +64,24 @@ endef
 
 $(eval $(call KernelPackage,bluetooth))
 
+define KernelPackage/btsdio
+  SUBMENU:=$(OTHER_MENU)
+  TITLE:=Bluetooth support
```

Comment:
We probably should mention SDIO in title.

Reply:
Correct, I'll fix that soon.